server/renderer.js

```diff
@@ -17,7 +17,7 @@ import {
   COOKIE_SHOW_ALL_CONTENT,
   LANG_UI_LANGUAGES,
   LANG_UKRAINIAN,
-  KC_BOT_USER_NAME,
+  KC_BOT_USER_NAME
```

Comment:
In future changes, PLEASE separate the syntax/lint issues to a different PR, it makes things harder.